fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OCFLPersistentStorageUtils.java

```diff
@@ -81,17 +81,22 @@ private OCFLPersistentStorageUtils() {
     /**
      * Returns the relative subpath of the resourceId based on the root object's resource id.
      *
-     * @param rootObjectId The fedora root object identifier
-     * @param resourceId   The identifier of the resource whose subpath you wish to resolve.
+     * @param rootFedoraObjectId The fedora root object identifier
+     * @param fedoraResourceId   The identifier of the resource whose subpath you wish to resolve.
      * @return The resolved subpath
      */
-    public static String relativizeSubpath(final String rootObjectId, final String resourceId) {
+    public static String relativizeSubpath(final String rootFedoraObjectId, final String fedoraResourceId) {
```

Comment:
Normally I'd expect a relative subpath to not include the parent, so given:
rootid = `info:fedora/my/root`
resource id = `info:fedora/my/root/some/more/stuff`
I'd expect the relative path to be `some/more/stuff`, but at the moment returns `root/some/more/stuff`. The former is how `Paths.relativize` works, although I understand not everything has to follow this model.

However, the result of this method appears to be passed directly into object session methods. If i'm understanding correctly, the session methods would be working with a subpath within the ocfl object that includes the last segment of the id of the root object, which I don't think is the structure we intended.

In other words, on disk I think we'd get 
`ocfl_root/my_root/v001/data/content/root/some/more/stuff`
whereas I would expect 
`ocfl_root/my_root/v001/data/content/some/more/stuff` 
since we can't assume that imported OCFL is going to put the contents of an object into a data subdirectory named after the ocfl object. I haven't tried running all the parts together to verify this though, this is just based off of my reading.

Also, `OCFLPersistentStorageUtils.resolveOCFLSubpath` currently only strips off the fcr:metadata suffix, but that already happens in `relativizeSubpath`. All calls of `resolveOCFLSubpath` receive the result of `relativizeSubpath`, so maybe we can drop `resolveOCFLSubpath` now?